_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md

```diff
@@ -24,21 +25,26 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to any set of any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characters) [accessible names][accessible name] that do not only consist of [whitespace](#whitespace).
+This rule applies to any set of any two or more HTML or SVG elements which
 
-**Note:** The test target for this rule is the full set of link elements that share the same [matching](#matching-characters) [accessible name][].
+- have the [semantic role][] of `link`, or a role that inherits from the `link` role; and
+- are in the same [set of browsing context related nodes][]; and
+- are [included in the accessibility tree][] of their respective [document][]; and
```

Comment:
I don't think you need to include this "of their respective document" part. At least, I don't see what it adds here.

Reply:
"**the accessibility tree**" is not correct since there is one per document, hence I felt the need for precision.
Replacing with "**an** accessibility tree" (which is less precise but probably good enough).